agent/xds/server_test.go

```diff
@@ -1,130 +1,95 @@
 package xds
 
 import (
-	"strings"
-	"sync"
 	"sync/atomic"
 	"testing"
 	"time"
 
+	envoy_api_v2 "github.com/envoyproxy/go-control-plane/envoy/api/v2"
 	envoy_discovery_v3 "github.com/envoyproxy/go-control-plane/envoy/service/discovery/v3"
 
+	"github.com/golang/protobuf/proto"
+	"github.com/golang/protobuf/ptypes"
+	"github.com/golang/protobuf/ptypes/any"
 	"github.com/stretchr/testify/require"
 	"google.golang.org/grpc/codes"
 	"google.golang.org/grpc/status"
 
 	"github.com/hashicorp/consul/acl"
-	"github.com/hashicorp/consul/agent/cache"
 	"github.com/hashicorp/consul/agent/proxycfg"
 	"github.com/hashicorp/consul/agent/structs"
-	"github.com/hashicorp/consul/sdk/testutil"
 )
 
-// testManager is a mock of proxycfg.Manager that's simpler to control for
-// testing. It also implements ConnectAuthz to allow control over authorization.
-type testManager struct {
```

Comment:
These moved and were enhanced so that the sotw and delta protocol tests could share more.